### PR TITLE
Draft: Adding Comments

### PR DESCRIPTION
**Pull request by Enough7**
Hi @riebl,
I will try to add some Code-Comments on this branch I already made locally.
This will help newcomers hopefully.

### CODE REVIEW

File: cmake/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+See [omnetpp/cmake: CMake module for OMNeT++ projects](https://github.com/omnetpp/cmake) and
+[AddOppRun — OMNeT++/CMake documentation](https://omnetpp.github.io/cmake/add-opp-run.html) for the origin of these files.
+They got adopted and adapted to artery.
```
Comment (riebl) [inline cmake/README.md:3]: Actually, it is the other way around: omnetpp/cmake emerged from Artery ;-)

File: src/artery/application/den/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+DEN stands for _Decentralized Environmental Notification_
```
Comment (riebl) [inline src/artery/application/den/README.md:1]: Maybe put that in a docs/glossary.md?

File: src/artery/ots/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+GTU is a Generic Traffic Unit, a term used by [OpenTrafficSim](https://github.com/averbraeck/opentrafficsim)
```
Comment (riebl) [inline src/artery/ots/README.md:1]: Again, glossary instead?

File: src/traci/sumo/foreign/tcpip/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Taken from [eclipse-sumo/sumo/src/foreign/tcpip](https://github.com/eclipse-sumo/sumo/tree/2f1d394a21aa4fc99f14dab1ea76c07fbb49ddf1/src/foreign/tcpip)
```
Comment (riebl) [inline src/traci/sumo/foreign/tcpip/README.md:1]: Redundant information, see src/traci/sumo/README.md

**Review by riebl — COMMENTED** [review]
I am not sure if putting READMEs everywhere is the best idea

**Comment by Enough7** [comment]
Hey, just to let you know I am currently a bit busy probably until February.

**Comment by Enough7** [comment]
I am sorry I probably will not have time in the near future to finish this pull request.

**Pull request closed without merging**